desktop-app/src/store/migrations.ts

```diff
@@ -88,4 +89,7 @@ export const migrations = {
       },
     ]);
   },
+  '1.2.2': (store: Store) => {
+    // Do we need a new migration???
```

Comment:
I don't think we need any migrations as we don't change any existing schema fields.

Reply:
Yes, leftover from early ideas. 
Removed.